diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/Makefile noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/Makefile
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/Makefile	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/Makefile	2018-08-14 14:16:40.601744161 +0200
@@ -209,7 +209,7 @@
 # Run make without -j if this happens.
 
 %.ttx: %.ttx.tmpl $(ADD_GLYPHS) $(ALL_COMPRESSED_FILES)
-	@python $(ADD_GLYPHS) -f "$<" -o "$@" -d "$(COMPRESSED_DIR)" $(ADD_GLYPHS_FLAGS)
+	@python3 $(ADD_GLYPHS) -f "$<" -o "$@" -d "$(COMPRESSED_DIR)" $(ADD_GLYPHS_FLAGS)
 
 %.ttf: %.ttx
 	@rm -f "$@"
@@ -217,8 +217,8 @@
 
 $(EMOJI).ttf: $(EMOJI).tmpl.ttf $(EMOJI_BUILDER) $(PUA_ADDER) \
 	$(ALL_COMPRESSED_FILES) | check_vs_adder
-	@python $(EMOJI_BUILDER) $(SMALL_METRICS) -V $< "$@" "$(COMPRESSED_DIR)/emoji_u"
-	@python $(PUA_ADDER) "$@" "$@-with-pua"
+	@python3 $(EMOJI_BUILDER) $(SMALL_METRICS) -V $< "$@" "$(COMPRESSED_DIR)/emoji_u"
+	@python3 $(PUA_ADDER) "$@" "$@-with-pua"
 	@$(VS_ADDER) -vs 2640 2642 2695 --dstdir '.' -o "$@-with-pua-varsel" "$@-with-pua"
 	@mv "$@-with-pua-varsel" "$@"
 	@rm "$@-with-pua"
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_aliases.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_aliases.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_aliases.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_aliases.py	2018-08-14 14:15:28.586483050 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2017 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_emoji_gsub.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_emoji_gsub.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_emoji_gsub.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_emoji_gsub.py	2018-08-14 14:15:28.586483050 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2014 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_glyphs.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_glyphs.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_glyphs.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_glyphs.py	2018-08-14 14:15:28.587483040 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 """Extend a ttx file with additional data.
 
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_svg_glyphs.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_svg_glyphs.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/add_svg_glyphs.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/add_svg_glyphs.py	2018-08-14 14:15:28.588483029 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright 2015 Google, Inc. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/check_emoji_sequences.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/check_emoji_sequences.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/check_emoji_sequences.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/check_emoji_sequences.py	2018-08-14 14:15:28.588483029 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2016 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/collect_emoji_svg.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/collect_emoji_svg.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/collect_emoji_svg.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/collect_emoji_svg.py	2018-08-14 14:15:28.589483019 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright 2015 Google, Inc. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/flag_glyph_name.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/flag_glyph_name.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/flag_glyph_name.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/flag_glyph_name.py	2018-08-14 14:15:28.589483019 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2014 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/flag_info.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/flag_info.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/flag_info.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/flag_info.py	2018-08-14 14:15:28.590483009 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 #
 # Copyright 2016 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/gen_version.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/gen_version.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/gen_version.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/gen_version.py	2018-08-14 14:15:28.590483009 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2015 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_html.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_html.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_html.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_html.py	2018-08-14 14:15:28.591482998 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2016 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_name_data.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_name_data.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_name_data.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_name_data.py	2018-08-14 14:15:28.591482998 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-#
 #
 # Copyright 2015 Google Inc. All rights reserved.
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_thumbnails.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_thumbnails.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_emoji_thumbnails.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_emoji_thumbnails.py	2018-08-14 14:15:28.592482988 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright 2017 Google Inc. All rights reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_test_html.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_test_html.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/generate_test_html.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/generate_test_html.py	2018-08-14 14:15:28.592482988 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright 2015 Google, Inc. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/map_pua_emoji.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/map_pua_emoji.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/map_pua_emoji.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/map_pua_emoji.py	2018-08-14 14:15:28.593482978 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2014 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/materialize_emoji_images.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/materialize_emoji_images.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/materialize_emoji_images.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/materialize_emoji_images.py	2018-08-14 14:15:28.593482978 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2016 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/strip_vs_from_filenames.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/strip_vs_from_filenames.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/strip_vs_from_filenames.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/strip_vs_from_filenames.py	2018-08-14 14:15:28.594482968 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright 2017 Google Inc. All rights reserved.
 #
diff -ru noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/svg_cleaner.py noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/svg_cleaner.py
--- noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362.orig/svg_cleaner.py	2018-08-10 22:25:19.000000000 +0200
+++ noto-emoji-3ffd20ec7504657aa17c84f9aa475ad09b20c362/svg_cleaner.py	2018-08-14 14:15:28.594482968 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright 2015 Google, Inc. All Rights Reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
