diff -ru noto-emoji-f09acc559b08e5f00c297c986d0e6112ebc88dbf.orig/Makefile noto-emoji-f09acc559b08e5f00c297c986d0e6112ebc88dbf/Makefile
--- noto-emoji-f09acc559b08e5f00c297c986d0e6112ebc88dbf.orig/Makefile	2019-07-08 18:40:44.000000000 +0200
+++ noto-emoji-f09acc559b08e5f00c297c986d0e6112ebc88dbf/Makefile	2019-07-09 10:07:13.028622382 +0200
@@ -15,7 +15,7 @@
 EMOJI = NotoColorEmoji
 font: $(EMOJI).ttf
 
-CFLAGS = -std=c99 -Wall -Wextra `pkg-config --cflags --libs cairo`
+CFLAGS = $(OPT_CFLAGS) `pkg-config --cflags --libs cairo`
 LDFLAGS = -lm `pkg-config --libs cairo`
 
 PNGQUANT = pngquant
@@ -101,6 +101,12 @@
 ALL_COMPRESSED_FILES = $(addprefix $(COMPRESSED_DIR)/, $(ALL_NAMES))
 
 # tool checks
+ifeq (,$(shell which $(PNGQUANT)))
+  ifeq (,$(wildcard $(PNGQUANT)))
+    MISSING_PNGQUANT = fail
+  endif
+endif
+
 ifeq (,$(shell which $(ZOPFLIPNG)))
   ifeq (,$(wildcard $(ZOPFLIPNG)))
     MISSING_ZOPFLI = fail
@@ -130,6 +136,11 @@
 
 compressed: $(ALL_COMPRESSED_FILES)
 
+check_quantization_tool:
+ifdef MISSING_PNGQUANT
+	$(error "$(PNGQUANT) is not available")
+endif
+
 check_compress_tool:
 ifdef MISSING_ZOPFLI
   ifdef MISSING_OPTIPNG
@@ -151,6 +162,8 @@
 	mkdir -p "$@"
 
 
+$(PNGQUANT): check_quantization_tool
+
 waveflag: waveflag.c
 	$(CC) $< -o $@ $(CFLAGS) $(LDFLAGS)
 
@@ -229,5 +242,5 @@
 .SECONDARY: $(EMOJI_FILES) $(FLAG_FILES) $(RESIZED_FLAG_FILES) $(RENAMED_FLAG_FILES) \
   $(ALL_QUANTIZED_FILES) $(ALL_COMPRESSED_FILES)
 
-.PHONY:	clean flags emoji renamed_flags quantized compressed check_compress_tool
+.PHONY:	clean flags emoji renamed_flags quantized compressed check_compress_tool check_quantization_tool
 
